src/graphlog.rs

```diff
@@ -558,30 +517,30 @@ mod tests {
     fn fork_multiple_chains() -> io::Result<()> {
         let mut buffer = vec![];
         let mut graph = AsciiGraphDrawer::new(&mut buffer);
-        graph.add_node(&10, &[Edge::direct(7)], b"o", b"node 10")?;
-        graph.add_node(&9, &[Edge::direct(6)], b"o", b"node 9")?;
-        graph.add_node(&8, &[Edge::direct(5)], b"o", b"node 8")?;
-        graph.add_node(&7, &[Edge::direct(4)], b"o", b"node 7")?;
-        graph.add_node(&6, &[Edge::direct(3)], b"o", b"node 6")?;
-        graph.add_node(&5, &[Edge::direct(2)], b"o", b"node 5")?;
-        graph.add_node(&4, &[Edge::direct(1)], b"o", b"node 4")?;
-        graph.add_node(&3, &[Edge::direct(1)], b"o", b"node 3")?;
-        graph.add_node(&2, &[Edge::direct(1)], b"o", b"node 2")?;
-        graph.add_node(&1, &[], b"o", b"node 1")?;
+        graph.add_node(&10, &[Edge::direct(7)], "⬤".as_bytes(), b"node 10")?;
+        graph.add_node(&9, &[Edge::direct(6)], "⬤".as_bytes(), b"node 9")?;
+        graph.add_node(&8, &[Edge::direct(5)], "⬤".as_bytes(), b"node 8")?;
+        graph.add_node(&7, &[Edge::direct(4)], "⬤".as_bytes(), b"node 7")?;
+        graph.add_node(&6, &[Edge::direct(3)], "⬤".as_bytes(), b"node 6")?;
+        graph.add_node(&5, &[Edge::direct(2)], "⬤".as_bytes(), b"node 5")?;
+        graph.add_node(&4, &[Edge::direct(1)], "⬤".as_bytes(), b"node 4")?;
+        graph.add_node(&3, &[Edge::direct(1)], "⬤".as_bytes(), b"node 3")?;
+        graph.add_node(&2, &[Edge::direct(1)], "⬤".as_bytes(), b"node 2")?;
+        graph.add_node(&1, &[], "⬤".as_bytes(), b"node 1")?;
 
         insta::assert_snapshot!(String::from_utf8_lossy(&buffer), @r###"
-        o node 10
-        | o node 9
-        | | o node 8
-        o | | node 7
-        | o | node 6
-        | | o node 5
-        o | | node 4
-        | o | node 3
-        |/ /  
-        | o node 2
-        |/  
-        o node 1
+        ⬤ node 10
+        │ ⬤ node 9
+        │ │ ⬤ node 8
+        ⬤ │ │ node 7
+        │ ⬤ │ node 6
+        │ │ ⬤ node 5
+        ⬤ │ │ node 4
+        │ ⬤ │ node 3
+        ├─╯ │ 
```

Comment:
Here would be more potential for saving space. The blank line is not strictly necessary.